test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -146,6 +155,15 @@ private static void destroyAndCheckProcess(final Process p, final ProcessBuilder
 	}
 
 	private static Process startProcess(final ProcessBuilder builder, final String name) throws IOException, InterruptedException {
+		// If a previous test did not terminate the process, kill it here (e.g. if assert got triggered before server was terminated)
+		final String commandLine = String.join(" ", builder.command());
+		final String pkillCommandLine[] = {"pkill", "-TERM", "-f", "-x", escapeCommandLineRegexChars(commandLine)};
```

Comment:
Is `pkill` available on all Linux platforms? How about zOS? We may want to support zOS at some point.
Is `startProcess` used for client, server or both?
"-f The pattern is normally only matched against the process name. When -f is set, the full command line is used. "
I can see the command line being used for server, but is that the case for the client as well?

Reply:
`startProcess` is used for both client and server. We always use the same command to start client/server processes so matching against the exact command should work for both.
I do not know what platforms `pkill` is available on, not sure how to find that info, but apparently regular `kill` is available on zOS https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxb200/termprocess.htm. We would need to find PID of the process first though.

Reply:
Note that we also use `pkill` to pause/resume server in `testServerUnreacheableForAWhile`.